lib/internal/modules/esm/loader.js

```diff
@@ -97,23 +99,11 @@ class ESMLoader {
   translators = getTranslators();
 
   constructor() {
-    if (getOptionValue('--experimental-loader').length > 0) {
-      emitExperimentalWarning('Custom ESM Loaders');
-    }
     if (getOptionValue('--experimental-network-imports')) {
       emitExperimentalWarning('Network Imports');
     }
   }
 
-  addCustomLoaders(userLoaders) {
-    const { Hooks } = require('internal/modules/esm/hooks');
-    this.#hooks = new Hooks(userLoaders);
-  }
-
-  preload() {
-    this.#hooks?.preload();
-  }
-
   async eval(
     source,
     url = pathToFileURL(`${process.cwd()}/[eval${++this.evalIndex}]`).href,
```

Comment:
Hey guys, @ljharb has reminded me in my implementation of the `registerLoader` function about windows support using `path.sep`.

```suggestion
    url = pathToFileURL(`${process.cwd()}${path.sep}[eval${++this.evalIndex}]`).href,
```

Reply:
It's unnecessary, forward slashes work everywhere.

Reply:
In this case yes - in the other PR, it was a path passed to dirname so i wasn’t sure.